### ISSUE TITLE
Overlay installation not report previous events?

### ISSUE THREAD

**Issue by ru1996** [Countly/countly-sdk-unity#96]
Hi, there. I am a Unity3D developer. I found that when I upgrade my app without uninstalling previous versions, the requests in the Queue which did not send last time will lose. 

I note that the data was not lost in the local DB file(db3.box).

Is this code not work?

**Comment by ZahidZafar** [comment]
Hello,
Please share the SDK version and platform you are using.

**Comment by ru1996** [comment]
countly-unity-sdk.20.11.0
both ios and android

**Comment by ZahidZafar** [comment]
Did you upgrade the SDK version too, or just upgrade the android or ios build?

**Comment by ru1996** [comment]
Just app.

**Comment by ru1996** [comment]
In fact, when the app is reinstalled, the unsent events stored locally will not be reported again.

**Comment by ZahidZafar** [comment]
Events and Views stored locally will send to the server when their count will be equal to or greater than the event threshold and Views threshold respectively. 
You can configure Events and Views threshold values in 'Countly prefab'. With session update call these events will send to the server.

**Comment by ru1996** [comment]
Yes, I know that. In general, countly works well. My problem is that when an overwrite installation occurs, countly did not resend the unsent requests previously stored in the db.
